src/pages/Proposal.vue

```diff
@@ -177,13 +291,38 @@ export default defineComponent({
 				};
 			}
 		},
-	},
-	setup() {
-		const formState = reactive(ProposalStore());
-		return {
-			formState,
-			validateMessages: VALIDATE_MESSAGES,
-		};
+		async optInLsigToApp(lsig: LogicSigAccount) {
+			try {
+				if (typeof this.daoStore.dao_id === "undefined") {
+					console.log("appId not defined");
+					return;
+				}
+				if (typeof this.daoStore.govt_id === "undefined") {
+					console.log("govt_id not defined");
+					return;
+				}
+				// fund lsig
+				await fundAmount(
+					this.walletStore.address,
+					lsig.address(),
+					2e6,
```

Comment:
What is 2e6 in this case? Let add more details in comments.

Reply:
To transfer 2 Algos in lsig account. Just to be sure if there is any execution that happens from lsig then we are not out of fund.